README.md

```diff
@@ -89,6 +91,52 @@ rescue Elastic::EnterpriseSearch::ClientException => e
 end
 ```
 
+### Permissions
+
+#### Listing all permissions
```

Comment:
I think it would be valuable to show how to use the method with paging parameters

Reply:
Yes, I agree. I must have missed that.

Reply:
Fixed in https://github.com/elastic/enterprise-search-ruby/pull/15/commits/1a415e40cce5f84941df1583802b46ee02f84a72